packages/denylist/wrangler.toml

```diff
@@ -35,7 +35,7 @@ routes = [
   { pattern = "denylist-staging.dag.haus/*", zone_id = "f2f8a5b1c557202c6e3d0ce0e98e4c8e" }
 ]
 kv_namespaces = [
-  { binding = "DENYLIST", id = "f4eb0eca32e14e28b643604a82e00cb3" }
+  { binding = "DENYLIST", id = "f4eb0eca32e14e28b643604a82e00cb3", preview_id = "f4eb0eca32e14e28b643604a82e00cb3" }
```

Comment:
this lets try out changes against the staging kv with

```
wrangler dev --remote --env staging
```

Reply:
oh nice, did not know this